apps/forklift/lib/forklift/init.ex

```diff
@@ -35,4 +46,16 @@ defmodule Forklift.Init do
       topic_subscriber_config: Application.get_env(:forklift, :topic_subscriber_config)
     ]
   end
+
+  defp initialize(args) do
+    Process.sleep(250)
```

Comment:
What’s this sleep get us if we still get a whole bunch at once? Can we throw some jitter in here?

Reply:
Don't know. Y'all put that sleep in here awhile back. I just moved it.

Reply:
:slow_clap:

Reply:
Thank you